### ISSUE TITLE
Dispatch Stats Page | Deprecate stats pages

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#12991]
## Description
The stats pages aren't being visited, and their code is old.  In addition, they are a source of accessibility audit violations.

## Acceptance criteria
- [ ] Remove stats pages
- [x] User sees a 404 message if they try to navigate to them

## Background/context/resources
GH issue for deprecating intake stats page:
https://github.com/department-of-veterans-affairs/caseflow/issues/11933

Sandra double checked in Google Analytics whether these pages were getting visits, and there were 0 visits since October 1st, 2019.

**Comment by msj0nes** [comment]
@Sjones352 is this ticket entirely complete? Can you close it if so.

**Comment by Sjones352** [comment]
@msj0nes I removed the Intake stat code from caseflow and disabled the Dispatch/ Certification code to return 404. This was done in phases as the it was a lot of files to remove. I will remove the rest of this code this sprint.
